UNRELEASED_CHANGELOG.md

```diff
@@ -75,6 +75,7 @@
 - Fixed attachments of camera. Now multiple videos and pictures can be taken from the camera.
 - Added the possibility to force light and dark theme. Set it in inside ChatUI to make all views, fragments and activity of the SDK light.
 - Fixed applying style to `SuggestionListView` when using it as a standalone component. You can modify the style using `suggestionListViewTheme` or `TransformStyle::suggestionListStyleTransformer`
+- Fixed markdown links rendering using custom linkify implementation.
```

Comment:
Let's mention this fix in old ui components module

Reply:
Done